keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -663,13 +674,13 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 
 #### Beta
 
-- Beta in 1.30: Since this feature is an extension of the external-resizer/external-provisioner usage flow, we are going to move this to beta with enhanced e2e and test coverage. Test cases are covered in sessions above: ``e2e tests``, ``Integration tests`` etc.
+- Beta in 1.31: Since this feature is an extension of the external-resizer/external-provisioner usage flow, we are going to move this to beta with enhanced e2e and test coverage. Test cases are covered in sessions above: ``e2e tests``, ``Integration tests`` etc.
```

Comment:
Do we need two beta stages to solve the beta AP not available by default? If so, we may need to mention it in the KEP.

cc @msau42

Reply:
If we will have 2 beta stages, I will change wording to the following.  

```
- Beta in 1.31 –  Feature gate on API Server enabled by default (Off by default in external-provisioner/external-resizer): Since this feature is an extension of the external-resizer/external-provisioner usage flow, we are going to move this to beta with enhanced e2e and test coverage. Test cases are covered in sessions above: ``e2e tests``, ``Integration tests`` etc. 
- Beta stage 2 in 1.3x – Feature gate enabled API Server **and** external-provisioner/external-resizer by default. 
```

Reply:
It's referring to this https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/3136-beta-apis-off-by-default

So the feature gates are on by default, but the API is off. This implies that the controllers need to handle this.

Reply:
Changing wording back. I assume the following is fine: 
```
- Beta in 1.31: Since this feature is an extension of the external-resizer/external-provisioner usage flow, we are going to move this to beta with enhanced e2e and test coverage. Test cases are covered in sessions above: ``e2e tests``, ``Integration tests`` etc. Controllers will handle VolumeAttributeClass feature gates being on by default, but beta API itself being disabled on clusters by default.
```